app.py

```diff
@@ -15,9 +15,100 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-"""This is the main script of the template project."""
+"""This is the main script of Thoth SLO reporter."""
+
+import os
+import logging
+import smtplib
+import pandas as pd
+
+from typing import Dict
+
+from prometheus_api_client import Metric, MetricsList, PrometheusConnect
+from prometheus_api_client.utils import parse_datetime, parse_timedelta
+from prometheus_client import CollectorRegistry, Gauge, push_to_gateway
+
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+from sli_metrics import SliMetricReport
+
+
```

Comment:
can we have __init__.py with a __version__ and a __service_version__ so that we can report it on startup to INFO log and so that kebechet can manage the releases? :)

Reply:
Added!